certbot/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
+*certbot-dns-google now depends on google-auth rather than oauth2client
```

Comment:
Certbot has had a new release (1.18) since you opened your PR.

Since the CHANGELOG file is changed, you will need to pull the `master` branch and merge it into your PR. Or rebase your PR on the updated `master` branch. Sorry! I know that's annoying.

Once you've done that, move the log entry into the 1.18 section.

Please also add a space between `*` and the change text.